_benchmark/user-guide/contributing-workloads.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: default
+title: Contributing custom workloads
+nav_order: 11
+parent: User guide
+---
+
+# Contributing custom workloads
```

Comment:
```suggestion
# Sharing custom workloads
```

Reply:
@IanHoang: Are you okay with this name change of do you prefer "contributing"?